committing.rst

```diff
@@ -218,14 +218,19 @@ the role is converted into a hyperlink. Example news entry with Sphinx::
 
 The inline Sphinx roles can be used to assist readers in finding more
 information and context on the changes made. If the inline link does not
-provide additional context, an inline reST code block should be used instead::
+provide additional context, an inline reST code block can be used instead::
 
    ``<object>``
 
 Before using any Sphinx roles, ensure that a corresponding entry exists
-within the documentation. Also, `build the HTML locally using make 
+within the documentation. When editing multiple news entries at once in
+"What's New" or Misc/NEWS.d/, `build the HTML locally using make 
 <https://devguide.python.org/documenting/#using-make-make-bat>`_ to
-verify that the link leads to the correct location.
+verify that the links lead to the correct locations. This does not need to be
+done for individual PR news entries.
+
+The Sphinx roles provide the maximum benefit in "What's New" entries,
+as they are read by the largest volume of users.
```

Comment:
>"Using sphinx roles is not strongly recommended. You don't have to be ashamed of using inline code blocks instead of roles."

I'm not at all opposed to adjusting this paragraph for clairity, but I respectfully disagree with that particular suggestion. In particular "Using sphinx roles is not strongly recommended", as this may end up having the effect of people thinking the feature is meaningless.

Instead, I think this would allow PR authors to use the inline code blocks if they prefer, without being dismissive of the potential usefulness of the roles:

"The Sphinx roles can be beneficial to readers, but they are not required. Inline code blocks can be used as a viable substitute."

Reply:
> "The Sphinx roles can be beneficial to readers, but they are not required. Inline code blocks can be used as a viable substitute."

This looks good to me.